docs/source/workflow/requests.md

```diff
@@ -11,7 +11,7 @@ Apollo Server accepts queries and mutations sent as `POST` requests. It also acc
 
 ## POST requests
 
-Apollo Server accepts `POST` requests with a JSON body. A valid request contains a `query` field, along with optional `variables` and an `operationName` (if `query` contains multiple possible operations). You must specify a `Content-Type` HTTP header with type `application/json`.
+Apollo Server accepts `POST` requests with a JSON body. A valid request contains a `query` field, along with optional `variables`, `extensions`, and an `operationName` (if `query` contains multiple possible operations). You must specify a `Content-Type` HTTP header with type `application/json`.
```

Comment:
```suggestion
Apollo Server accepts `POST` requests with a JSON body. A valid request contains a `query` field, along with optional `variables`, `extensions`, and an `operationName` (which disambiguates if `query` contains multiple operations). You must specify a `Content-Type` HTTP header with type `application/json`.
```

I think the previous (and prior to your PR) wording may have inaccurately implied that `operationName` is only legal if `query` contains multiple operations.

Reply:
You raise a good point but I think `disambiguates` feels a little forced here, how does this feel:
> Apollo Server accepts `POST` requests with a JSON body. A valid request contains a `query` field, along with optional `variables`, `extensions`, and an `operationName` (clarifying which `query` to run if the request contains multiple possible operations).